conf/test.config

```diff
@@ -26,4 +26,15 @@ params {
     mindagap_loopnum = 100
     clahe_pyramid_tile = 368
     segmentation_method = "mesmer,cellpose"
+
+    // Only used if additionally params.create_training_subset is used
+    crop_size_x = 30
+    crop_size_y = 30
+    crop_amount = 2
+}
+
+process {
+    withName: 'MULTIQC' {
+        container = "community.wave.seqera.io/library/multiqc:1.24.1--789bc3917c8666da"
+    }
```

Comment:
I don't think that this configuration should be added to test.config, even if it's for development in gitpod. If you want to develop in gitpod and this is causing issues, I would suggest adding this to a small custom nextflow.config file that you can add to tests, it's not a long snippet anyways...

```suggestion

```